app.rb

```diff
@@ -0,0 +1,191 @@
+# rubocop:disable Metrics
+require './person'
+require './capitalize_decorator'
+require './trimmer_decorator'
+require './student'
+require './classroom'
+require './teacher'
+require './rental'
+require './book'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    # @class = Classroom.new('Grade 5')
+  end
+
+  def run
+    puts 'Welcome To School Library App!'
+    puts
+    puts
+    message_menu
+  end
+
+  def message_menu
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
```

Comment:
- [ ] Hello, excellent job on adding the UI, but as the requirement say that the `App.rb` will contain the logic and the  `main.rb` will have the start app with the interface function in the app.rb file

Reply:
May I please call you on zoom so that you can help me understand the concept?